carparkmanager-persistence/src/main/java/enums/ClassificationOfEmployee.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package enums;
+
+/**
+ *
+ * @author xbonco1
+ */
+public enum ClassificationOfEmployee {
```

Comment:
neni to pravidlo, ale byva uzitocne, ked je z nazvu poznat, ze ide o enum, napr.  ClassificationOfEmployeeEnum